dcc-portal-ui/app/scripts/software/js/software.js

```diff
@@ -43,9 +43,13 @@
       Page.setTitle('Software Downloads');
 
       $scope.artifactId = 'icgc-storage-client';
+      $scope.icgcGetArtifactId = 'icgc-get';
```

Comment:
I think these scope variables should be cleaned up. Lets use an object for each software:

```javascript
$scope.icgcGet = {
  artifactId: 'icgc-get'
  ...
};
$scope.icgcStorageClient = {
  artifactId: 'icgc-storage-client'
  ...
};
```